zebra-rpc/src/methods.rs

```diff
@@ -869,6 +994,22 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 #[derive(Copy, Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
 
+/// Response to a `z_gettreestate` RPC request.
+///
+/// Contains the hex-encoded Sapling & Orchard note commitment trees, and their
+/// corresponding [`block::Hash`], [`Height`], and block time.
+#[derive(serde::Serialize)]
```

Comment:
Optional: we usually derive these traits to make testing easier.

```suggestion
#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize)]
```